test/server/cards/03_TWI/leaders/MaulARivalInDarkness.spec.ts

```diff
@@ -0,0 +1,59 @@
+describe('Maul, A Rival In Darkness', function() {
+    integration(function(contextRef) {
+        describe('Maul, A Rival In Darkness\'s undeployed leader ability', function() {
+            beforeEach(function() {
+                contextRef.setupTest({
```

Comment:
Since there's only one test case here you could remove the `beforeEach` and the `describe` and just merge everything into one test case that has the `setupTest` at the beginning (see Zillo Beast tests for an example). Same for the deployed test